extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/convert/AwsElasticBlockStoreVolumeConverter.java

```diff
@@ -0,0 +1,24 @@
+
+package io.quarkus.kubernetes.deployment.convert;
+
+import java.util.Map;
+
+import io.dekorate.kubernetes.config.AwsElasticBlockStoreVolume;
+import io.dekorate.kubernetes.config.AwsElasticBlockStoreVolumeBuilder;
+import io.quarkus.kubernetes.deployment.config.AwsElasticBlockStoreVolumeConfig;
+
+public class AwsElasticBlockStoreVolumeConverter {
```

Comment:
I assume this are the converters that you would like to autogenerate in a future iteration?

Reply:
Yeah